tests/02_eradiate/01_unit/radprops/test_rayleigh_scattering.py

```diff
@@ -22,7 +22,7 @@ def test_sigma_s_air():
     # Compare to reference value computed from scattering cross section in
     # Bates (1984) Planetary and Space Science, Volume 32, No. 6.
     assert np.allclose(
-        compute_sigma_s_air(number_density=_LOSCHMIDT), expected, rtol=1e-2
+        expected, compute_sigma_s_air(number_density=_LOSCHMIDT), rtol=1e-2
```

Comment:
what  is going on wit the `allclose` statements? You mentioned a regression in DrJit, is this a mitigation for that?

Reply:
[`allclose()` is asymmetric](https://numpy.org/doc/stable/reference/generated/numpy.allclose.html): one of the two arguments is the reference. Now, looking again at the formula, I'm realising that it's `b`: contrary to what I changed here, the initial formula was correct. There are howver other parts of the code where things are incorrect, I'll fix that (and revert the inappropriate changes I made).

This is unrelated with Dr.Jit or Mitsuba.

Reply:
Done. Turns out the tests were a mess with that respect. This is fixed now (wherever the concept of a reference can be relevance, of course).